vscode-powershell.build.ps1

```diff
@@ -119,20 +134,30 @@ task UpdateReadme {
     $readmeContent = Get-Content -Path $readmePath
     if (!($readmeContent -match "This is the PREVIEW version of the PowerShell extension")) {
         $readmeContent[0] = $newReadmeTop
-        $readmeContent > $readmePath
+        $readmeContent | Set-Content $readmePath -Encoding utf8
     }
 }
 
 task UpdatePackageJson {
-    $script:PackageJson.name = "powershell-preview"
-    $script:PackageJson.displayName = "PowerShell Preview"
-    $script:PackageJson.description = "(Preview) Develop PowerShell scripts in Visual Studio Code!"
-    $script:PackageJson.preview = $true
-    $script:ExtensionName = $script:PackageJson.name
-    Set-Content -Path $PSScriptRoot/package.json ($script:PackageJson | ConvertTo-Json -Depth 100)
+    if ($script:PackageJson.version -like "*preview*") {
+        $script:PackageJson.name = "powershell-preview"
+        $script:PackageJson.displayName = "PowerShell Preview"
+        $script:PackageJson.description = "(Preview) Develop PowerShell scripts in Visual Studio Code!"
+        $script:PackageJson.preview = $true
+    } else {
+        $script:PackageJson.name = "powershell"
+        $script:PackageJson.displayName = "PowerShell"
+        $script:PackageJson.description = "Develop PowerShell scripts in Visual Studio Code!"
+        $script:PackageJson.preview = $false
+    }
+
+    $revision = if ($env:VSTS_BUILD_VERSION) { $env:VSTS_BUILD_VERSION } else { 9999 }
```

Comment:
Thoughts on using `$env:BUILD_BUILDID` instead of this `$env:VSTS_BUILD_VERSION`?

@rjmholt

Reply:
There's also `System.PullRequest.PullRequestId` but I don't think that'll be defined in CI builds, only PR builds

Reply:
```suggestion
    $revision = if ($env:BUILD_BUILDID) { $env:BUILD_BUILDID } else { 9999 }
```